### ISSUE TITLE
question: perhaps we should relate gist to CCO?

### ISSUE THREAD

**Issue by justin2004** [semanticarts/gistBFO#15]
each time i've had to use BFO it has been because i had to use [CCO](https://github.com/CommonCoreOntology/CommonCoreOntologies) (which descends from BFO).

CCO is situated conceptually near where gist is conceptually. 

you can see how big the conceptual leaps (from gist to BFO) are in this repo:
gist:Intention subclassof bfo:continuant.

whereas if we did something like this:
gist:Intention subclassof cco:DirectiveInformationContentEntity
that is saying something with more specificity.

gist:Intention has subclasses like Function and Specification and cco:DirectiveInformationContentEntity has subclasses like ArtifactFunctionSpecification, Objective, Plan -- so their sublasses are quite similar.

also i think CCO is becoming the defacto BFO descended mid level ontology.

**Comment by justin2004** [comment]
cc
@johnbeve

**Comment by kstudzin** [comment]
Absolutely! We are working on a CCO mapping! (It will probably be in it's own repository)
